docs/lang/articles/advanced/sparse.md

```diff
@@ -2,100 +2,116 @@
 sidebar_position: 3
 ---
 
-# Sparse computation
-
-Compiler-level support for spatially sparse computation is a unique feature of Taichi.
-
-![image](https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/sparse_grids.gif)
-
-Figure: A swinging "Taichi" pattern represented with a 512x512 sparse grid. The sparse grid has a multi-level *tree* structure.
-White stands for inactive tree nodes, and active tree nodes are darker.
-
-The sparse grid above has the following structure:
-- The grid is divided into 8x8 `block1` containers;
-- Each `block1` container has 4x4 `block2` cells;
-- Each `block2` container has 4x4 `block3` cells;
-- Each `block3` container has 4x4 pixel cells;
-- Each pixel contains an `i32` value `x[i, j]`.
+# Sparse spatial data structures
```

Comment:
Shall we consistently use "Sparse spatial data structures" in this doc when we refer to this term?

Reply:
Yes, good point.